docs/advanced/other_advanced.rst

```diff
@@ -14,8 +14,10 @@ They make use of a pandas dataframe ``df`` containing dummy data, that can be lo
     df = get_dummy_data()
 
 .. note::
-   It is not recommended to copy and paste code from this page to reproduce the shown examples.
-   Instead, please visit the `plot gallery <https://plothist.readthedocs.io/en/latest/example_gallery/>`_, because it contains a standalone script for each example, that you can run directly to produce the plot.
+   This page presents functions of ``plothist`` step by step and gives information about how to use them.
+   It is not necessary to copy and paste code from this page to reproduce the shown examples.
+   To reproduce the examples, please visit the `plot gallery <https://plothist.readthedocs.io/en/latest/example_gallery/>`_, because it contains a standalone script for each example, that you can run directly.
```

Comment:
also in this section, the only thing missing from the scripts is the `get_data` line, otherwise the full script is in the doc. This page presents how to use functions in complex examples.